modules/builds-chaining-builds.adoc

```diff
@@ -23,7 +23,7 @@ The first build takes the application source and produces an image containing a
 
 [source,yaml]
 ----
-apiVersion: v1
+apiVersion: build.openshift.io/v1
```

Comment:
@adambkaplan ptal for build/image changes

Reply:
lgtm